user_sync/config.py

```diff
@@ -333,7 +312,14 @@ def get_rule_options(self):
         if (new_account_type == None):
             new_account_type = user_sync.identity_type.ENTERPRISE_IDENTITY_TYPE
             self.logger.warning("Assuming the identity type for users is: %s", new_account_type)
-        
+
+        max_deletions_per_run = 10
+        max_missing_users = 200
+        if self.options['remove_nonexistent_users'] == True:
```

Comment:
I think you may need to remove this check.     I recall there is code after the initialization to check what keys are not used in the config and error out.   If there is a condition here, the config file will have an unused limits section when remove_nonexistent_users is false.